frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -87,7 +87,7 @@ const Metric = (props: MetricProps) => {
     }, [props.colorMap, props.max]);
 
     const data = useMemo(() => {
-        const mode = props.type === "none" ? [] : ["gauge"];
+        const mode = props.type == null || (typeof props.type === "string" && props.type.toLowerCase() === "none") ? [] : ["gauge"];
```

Comment:
you need to declare type as string|null

Reply:
Hey Fred, I might have misunderstood 😅. Is this what you req. to change?

Reply:
I'd like you to change the definition of the type attribute in the MetricsProps interface so that the test can pass